lib/Controller/MessageApiController.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+declare(strict_types=1);
+/**
+ * SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-only
```

Comment:
```suggestion
 * SPDX-License-Identifier: AGPL-3.0-or-later
```

Reply:
will do that in the other PR (so I don't have to rebase this one again)